crates/apub/src/objects/mod.rs

```diff
@@ -70,6 +70,25 @@ pub trait FromApubToForm<ApubType> {
     Self: Sized;
 }
 
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+pub enum MediaTypeMarkdown {
+  #[serde(rename = "text/markdown")]
+  Markdown,
+}
+
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+pub enum MediaTypeHtml {
+  #[serde(rename = "text/html")]
+  Markdown,
+}
+
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Source {
+  content: String,
+  media_type: MediaTypeMarkdown,
+}
```

Comment:
What's this mediatype about, and why are there two different enums instead of one `MediaType { Markdown, Html }`

Reply:
I added documentation to explain that, please read it and tell me if it is still unclear.